services/QuillLMS/app/models/user.rb

```diff
@@ -164,6 +164,16 @@ class User < ActiveRecord::Base
 
   attr_accessor :newsletter
 
+  def self.deleted_users
+    where(
+      <<-SQL
+        name LIKE 'Deleted_User_%'
+          AND email LIKE 'deleted_user_%'
+          AND username LIKE 'deleted_user_%'
+      SQL
+    )
+  end
```

Comment:
This has got to be a pretty expensive query given multiple `like`s.  Not suggesting a change here, but I am wondering if we should be thinking about a better solution in the future.

Reply:
I didn't notice any performance issues while running the rake task on staging.  My understanding is that since the query is a conjunction on the same table, the result set gets smaller and smaller after each `like`, therefore we're not doing a full scan of the table each time.

That being said, I think it would be a good idea to redesign the deleted user representation in the database as hard-coded deleting_user on to attribute values is definitely a code smell.